src/loading.jl

```diff
@@ -0,0 +1,92 @@
+loadleaf!(dst, src, err) = dst
```

Comment:
I also wonder if there should be more errors here:
```suggestion
loadleaf!(dst, src, err) = dst
loadleaf!(dst::AbstractArray, src, err) = error()
loadleaf!(dst, src::AbstractArray, err) = error()
```